client/src/holdings/AddTradeDialog.tsx

```diff
@@ -86,7 +86,7 @@ export default function AddTradeDialog({
           }}>
           {({isSubmitting}) => (
             <Form>
-              <DialogTitle id='form-dialog-title'>Add Trade</DialogTitle>
+                <DialogTitle id='form-dialog-title'>Add Trade ({holding.symbol})</DialogTitle>
```

Comment:
Nice one, just a small thing, you've added an extra indent where there wasn't one before, so assuming this wasn't intended, can you remove it. Then it's good to go. You can do

```sh
git commit --amend
git push -f origin
```

To amend your existing commit

Reply:
Should be done

Reply:
Still looks the same to me, there should be no indentation change from what was there before (unless the autoformatter puts the code on multiple lines). Do you have the 'format on save' on in VS Code? That should fix it. See here: https://github.com/willbeaufoy/portfolio-tracker#code-editor